pootle/apps/reports/views.py

```diff
@@ -501,38 +503,43 @@ def get_daily_activity(user, scores, start, end):
     current_day_score = 0
     translated_group = {}
     reviewed_group = {}
+    suggested_group = {}
     for score in scores:
         score_time = make_naive(score.creation_time)
         date = score_time.date()
 
         translated, reviewed = score.get_paid_wordcounts()
-        if translated is not None or reviewed is not None:
+        suggested = score.get_suggested_wordcount()
+
+        if any(map(lambda x: x is not None, [translated, reviewed, suggested])):
```

Comment:
Looks nicer, but I find hard to immediately understand what we are checking. Sometimes explicit is better than explicit.